teps/0116-referencing-finally-task-results-in-pipeline-results.md

```diff
@@ -0,0 +1,87 @@
+---
+status: proposed
```

Comment:
@vsinghai please mark this as implementable as the design is complete

Reply:
done